electrumsv/wallet_database/types.py

```diff
@@ -375,14 +375,25 @@ class TransactionOutputSpendableRow(NamedTuple):
     derivation_type: DerivationType | None   # Spendable type field.
     derivation_data2: bytes | None           # Spendable type field.
 
-    # def to_account_row(self) -> AccountTransactionOutputSpendableRow:
-    #     assert self.account_id is not None
-    #     assert self.derivation_type is not None
-    #     return AccountTransactionOutputSpendableRow(tx_hash=self.tx_hash,
-    #           txo_index=self.txo_index,
-    #         value=self.value, keyinstance_id=self.keyinstance_id, script_type=self.script_type,
-    #         flags=self.flags, account_id=self.account_id, masterkey_id=self.masterkey_id,
-    #         derivation_type=self.derivation_type, derivation_data2=self.derivation_data2)
+
+class TransactionOutputSpendRow(NamedTuple):
+    """
+    Extended version of `TransactionOutputSpendableRow` with `txi_index` field.
+    """
+    # Transaction input fields.
+    txi_index: int
```

Comment:
This `txi_index` field seems confusing to me... Why would it not include both the input tx_hash and the input index together? Seems weird to only include the `txi_index` on its own. How is that useful for anything?

Not saying it's necessarily wrong - more that it's taking me a bit of effort to understand this part...

Reply:
Ah. Never mind. I get it now.